src/components/Input/FileInput.tsx

```diff
@@ -1,25 +1,19 @@
 import React, { ChangeEvent } from 'react';
 import Input from './Input';
 
-type FileInputProps = {
+interface FileInputProps extends Omit<React.ComponentProps<typeof Input>, 'type'> {
   onChange: (files: FileList | null) => void;
-  type: 'file';
-};
+}
 
-class FileInput extends React.Component<FileInputProps> {
-  handleChange = (changeEvent: ChangeEvent) => {
-    const files = (changeEvent.target as HTMLInputElement).files;
-    if (this.props.onChange) {
-      this.props.onChange(files);
-    }
-  };
+const FileInput = (props: FileInputProps) => {
+  const { onChange } = props;
 
-  render() {
-    /* eslint-disable-next-line  @typescript-eslint/no-unused-vars -- Let's figure out a better way to omit props for this scenario */
-    const { type, onChange, ...props } = this.props;
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const files = event.target.files;
+    onChange?.(files);
+  };
 
-    return <Input type="file" onChange={this.handleChange} {...props} />;
-  }
-}
+  return <Input type="file" onChange={handleChange} />;
```

Comment:
Hey @JeremyRH with a functional component, how do you recommend I include this spread operator for props? `{...props}`